joomla/joomla-cms
# Fix MFA Authentication Code by Email plugin's Force Enable feature

Pull Request for issue reported in https://github.com/joomla/joomla-cms/pull/37912#issuecomment-1152577174

### Summary of Changes

The MFA plugin's “Force Enable” option didn't do anything. This PR addresses that.

### Testing Instructions

* Set up MFa for your user account using any method _except_ ‘Code by Email’.
* Edit the ‘Multi-factor Authentication - Authentication Code by Email’ plugin
* Set Force Enable to Yes
* Save & Close
* Log out
* Log back in
* When you see the MFA page click choose another method

### Actual result BEFORE applying this Pull Request

You do not see an “Authentication Code by Email” option

### Expected result AFTER applying this Pull Request

You see an (automatically added) “Authentication Code by Email” option.

### Documentation Changes Required

None
